wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/WorkflowDefinitionElementToWomWorkflowDefinition.scala

```diff
@@ -33,6 +34,14 @@ object WorkflowDefinitionElementToWomWorkflowDefinition extends Util {
 
     val a: WorkflowDefinitionConvertInputs = eliminateInputDependencies(b)
 
+    val (meta, parameterMeta) = processMetaSections(a.definitionElement.metaSection, a.definitionElement.parameterMetaSection)
+
+    val allowNestedInputs: Boolean = {
+      meta.get("allowNestedInputs").flatMap {
+        case x: MetaValueElementBoolean => Option(x.value)
+        case _ => None
```

Comment:
TOL (Talking Out Loud, not gating approval of this PR): Is it the case that specifying

```
   meta {allowNestedInputs: "I like turtles"}
```
would silently set `allowNestedInputs` to `false`? Should there be a warning or error if the wrong type (something other than `MetaValueElementBoolean`) was specified?

Reply:
```meta {allowNestedInputs: "I like turtles"}``` Will indeed set `allowNestedInputs` to false. I see no problem here, as it is clearly not the intention of the WDL writer to enable a feature.
Booleans are always `true` or `false`. I do not think extra code is needed to evaluate cases such as `"true"`, `"false"`, `0`, `1` etc. If people start complaining at this behavior we can easily change it later. For now, I see no reason for the extra effort and code complexity (also I am lazy :wink:).